.github/workflows/generate-e2e-matrix.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
if we go this way I would move this file to the level above since it's not a workflow file

Reply:
I think we can just add the array in the workflow file to the matrix part and adding an include for the composer v1 e2e test.

Reply:
Done